### PR TITLE
Introduce tool abrt-dump-journal-xorg

### PR DESCRIPTION
**Pull request by mhabrnal**
Resolves abrt/abrt#1023
Related to rhbz#1246373

### CODE REVIEW

**Comment by sorki** [comment]
Typo in commit message: 'xorg: introduce library xort-utils'

File: tests/runtests/journal-xorg-crash-processing/runtest.sh [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    fi
+
+    prepare
+
+    rlRun "journalctl --flush"
+
+    rlRun "ABRT_DUMP_JOURNAL_XORG_DEBUG_FILTER=\"SYSLOG_IDENTIFIER=${SYSLOG_IDENTIFIER}\" setsid ${exe} ${args} -vvv -f -xD -o >${crash_name}.log 2>&1 &"
+    rlRun "ABRT_DUMPER_PID=$!"
+cp ${crash_name}.log /tmp/test.log
+
```
Comment (sorki) [inline tests/runtests/journal-xorg-crash-processing/runtest.sh]: missing spaces

File: tests/runtests/journal-xorg-crash-processing/runtest.sh [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    rlRun "killall -TERM abrt-dump-journal-xorg"
+    sleep 2
+
+    if [ -d /proc/$ABRT_DUMPER_PID ]; then
+        rlLogError "Failed to kill the abrt journal xorg dumper"
+        rlRun "kill -TERM -$ABRT_DUMPER_PID"
+    fi
+
+    grep -v "abrt-dump-journal-xorg" ${crash_name}".log" > ${crash_name}".log.right"
+cp -v ${crash_name}".log.right" /tmp/
```
Comment (sorki) [inline tests/runtests/journal-xorg-crash-processing/runtest.sh]: spaces ^^

**Comment by sorki** [comment]
Otherwise fine. Good job

**Comment by mhabrnal** [comment]
I've addressed all @sorki 's comments.

**Comment by jfilak** [comment]
Thank you!

**Pull request merged**